.run_cdc_dev.env

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Env variables for running run_cdc_dev.sh
+
+# The 3 keys / passwords below are not checked in.
+# Either uncomment these lines and set them here or set them separately as environment variables.
+DC_API_KEY=
+MAPS_API_KEY=
+DB_PASS=
+
+# Developers will typically change the Cloud SQL and GCS folders below when developing against various DB instances and embeddings files.
+# So keep them at the top of the file.
+
+# Cloud SQL settings.
+# Set to false is not using Cloud SQL.
+USE_CLOUDSQL=true
+CLOUDSQL_INSTANCE=datcom-website-dev:us-central1:dc-dev
+DB_USER=root
+
+# GCS or local folder to "user data" folder.
+# This is primarily for NL embeddings.
+# The code looks for it under $USER_DATA_PATH/datacommons/nl/custom_embeddings.yaml
+USER_DATA_PATH=gs://customdc-data/dc-dev
+GOOGLE_CLOUD_PROJECT=datcom-website-dev
```

Comment:
nit: put GOOGLE_CLOUD_PROJECT above line 28

Reply:
does this have anything to do with CLOUDSQL? if it's only for GCS maybe say GCS_PROJECT?

Reply:
The gcloud sdk looks for this variable so not something we can change.